mmdet3d/core/visualizer/show_result.py

```diff
@@ -81,6 +79,12 @@ def show_result(points, gt_bboxes, pred_bboxes, out_dir, filename, show=True):
         filename (str): Filename of the current frame.
         show (bool): Visualize the results online.
     """
+    try:
```

Comment:
This place do not need to do that as the warning should be bumped in open3d_vis.py